assets/queries/cloudFormation/filter_and_alarm_exist_for_unauthorized_api_calls/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Filter_And_Alarm_Exist_For_Unauthorized_API_Calls",
+  "queryName": "Filter And Alarm Not Exists For Unauthorized API Calls",
```

Comment:
change query name to "Filter And Alarm Do Not Exists For Unauthorized API Calls"

Reply:
Done